django/core/mail/__init__.py

```diff
@@ -91,6 +91,8 @@ def mail_admins(subject, message, fail_silently=False, connection=None,
     """Send a message to the admins, as defined by the ADMINS setting."""
     if not settings.ADMINS:
         return
+    if not all(isinstance(a, tuple) and len(a) == 2 for a in settings.ADMINS):
```

Comment:
Probably we should also allow for a `list` to do not create any regression, and raise:
```python
raise ValueError(
    'The ADMINS setting must be a list or tuple containing two values.'
)
```